commands/command_env.go

```diff
@@ -16,7 +16,7 @@ var (
 func envCommand(cmd *cobra.Command, args []string) {
 	lfs.ShowConfigWarnings = true
 	config := lfs.Config
-	endpoint := config.Endpoint()
+	endpoint := config.Endpoint("download")
```

Comment:
Does it make sense to setup constants like `DownloadOperation` instead of passing in a string?